libc/src/stdbit/CMakeLists.txt

```diff
@@ -1,99 +1,19 @@
-add_entrypoint_object(
-  stdc_leading_zeros_uc
-  SRCS
-    stdc_leading_zeros_uc.cpp
-  HDRS
-    stdc_leading_zeros_uc.h
-  DEPENDS
-    libc.src.__support.CPP.bit
-)
-
-add_entrypoint_object(
-  stdc_leading_zeros_us
-  SRCS
-    stdc_leading_zeros_us.cpp
-  HDRS
-    stdc_leading_zeros_us.h
-  DEPENDS
-    libc.src.__support.CPP.bit
-)
-
-add_entrypoint_object(
-  stdc_leading_zeros_ui
-  SRCS
-    stdc_leading_zeros_ui.cpp
-  HDRS
-    stdc_leading_zeros_ui.h
-  DEPENDS
-    libc.src.__support.CPP.bit
-)
-
-add_entrypoint_object(
-  stdc_leading_zeros_ul
-  SRCS
-    stdc_leading_zeros_ul.cpp
-  HDRS
-    stdc_leading_zeros_ul.h
-  DEPENDS
-    libc.src.__support.CPP.bit
-)
-
-add_entrypoint_object(
-  stdc_leading_zeros_ull
-  SRCS
-    stdc_leading_zeros_ull.cpp
-  HDRS
-    stdc_leading_zeros_ull.h
-  DEPENDS
-    libc.src.__support.CPP.bit
-)
-
-add_entrypoint_object(
-  stdc_leading_ones_uc
-  SRCS
-    stdc_leading_ones_uc.cpp
-  HDRS
-    stdc_leading_ones_uc.h
-  DEPENDS
-    libc.src.__support.CPP.bit
-)
-
-add_entrypoint_object(
-  stdc_leading_ones_us
-  SRCS
-    stdc_leading_ones_us.cpp
-  HDRS
-    stdc_leading_ones_us.h
-  DEPENDS
-    libc.src.__support.CPP.bit
-)
-
-add_entrypoint_object(
-  stdc_leading_ones_ui
-  SRCS
-    stdc_leading_ones_ui.cpp
-  HDRS
-    stdc_leading_ones_ui.h
-  DEPENDS
-    libc.src.__support.CPP.bit
-)
-
-add_entrypoint_object(
-  stdc_leading_ones_ul
-  SRCS
-    stdc_leading_ones_ul.cpp
-  HDRS
-    stdc_leading_ones_ul.h
-  DEPENDS
-    libc.src.__support.CPP.bit
-)
-
-add_entrypoint_object(
-  stdc_leading_ones_ull
-  SRCS
-    stdc_leading_ones_ull.cpp
-  HDRS
-    stdc_leading_ones_ull.h
-  DEPENDS
-    libc.src.__support.CPP.bit
-)
+list(APPEND prefixes
```

Comment:
I think you should set(prefixes ...) and set(suffixes ...) instead of appending, just in case they were set somewhere before.

Reply:
done in 5d94dd33e688 PTAL